library/styleguideComponents/StyleGuide/index.js

```diff
@@ -38,6 +39,7 @@ export default class StyleGuide extends React.Component {
                     {toc}
                 </aside>
                 <main className="react-rainbow-main-content">
+                    <ProjectSelector />
```

Comment:
Why ProjectSelector inside main?
I think that it can be placed the most outer possible.